pkgs/tools/filesystems/rmount/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchgit, makeWrapper, lib, pkgs ? import <nixpkgs> {} }:
+with pkgs;
+
+stdenv.mkDerivation rec {
+  name    = "rmount-${version}";
+  version = "1.0.1";
+  rev = "v${version}";
+
+  src = fetchgit {
+    inherit rev;
+    url    = "https://github.com/Luis-Hebendanz/rmount";
+    sha256 = "1wjmfvbsq3126z51f2ivj85cjmkrzdm2acqsiyqs57qga2g6w5p9";
+  };
+
+  buildInputs = [ stdenv makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/share/man/man1
+    cp ${src}/rmount.man $out/share/man/man1/rmount.1
+    cp ${src}/rmount.bash $out/bin/rmount-noenv
+    cp ${src}/config.json $out/share/config.json
+    chmod +x $out/bin/rmount-noenv
+
+    makeWrapper $out/bin/rmount-noenv $out/bin/rmount \
+    --prefix PATH : ${lib.makeBinPath [ nmap jq cifs-utils sshfs ]}
```

Comment:
[From `makeWrapper`'s documentation](https://nixos.org/nixpkgs/manual/#ssec-stdenv-functions):

> `wrapProgram` is a convenience function you probably want to use most of the time.

In this particular instance, unless it is expected for a end-user to use `rmount-noenv`, you probably would prefer to use `wrapProgram`.

> `wrapProgram` *`executable`* *`makeWrapperArgs`*
> Convenience function for makeWrapper that automatically creates a sane wrapper file It takes all the same arguments as `makeWrapper`, except for `--argv0`.